GitHub-Repositories-Repository/github-repositories-repository.json

```diff
@@ -273,8 +277,8 @@
     "/properties/SecurityAndAnalysis"
   ],
   "primaryIdentifier": [
-    "/properties/Owner",
-    "/properties/Name"
+    "/properties/Id",
+    "/properties/Owner"
```

Comment:
Is Owner still part of the primary id?  It looks like it isn't based on the changes to handlers

Reply:
`owner` is still used in list handler:
```
const requestRoute = model.owner
            ? 'GET /orgs/{org}/repos'
            : 'GET /user/repos';
```
I assume that therefore it is required to remain as primaryIdentifier? Feel free to correct me if I am wrong :)

Reply:
Is it possible to add it to `additionalIdentifiers` instead?